src/sentry/static/sentry/app/components/asyncComponent.tsx

```diff
@@ -341,6 +349,10 @@ export default class AsyncComponent<
     this.onRequestError(error, args);
   }
 
+  onEndpointsLoadedSuccessfully() {
```

Comment:
Can we move this to group with the other `on*` methods? Maybe `onLoadAllEndpointsSuccess` to match naming of the other methods?